scipy/stats/_stats_mstats_common.py

```diff
@@ -3,10 +3,9 @@
 import numpy as np
 
 from . import distributions
-from . import futil
 
 
-__all__ = ['find_repeats', 'linregress', 'theilslopes']
+__all__ = ['_find_repeats', 'linregress', 'theilslopes']
```

Comment:
It has an underscore because it's a helper for a routine with the same name -- we're not really consistent in private routine naming. And I don't think `__all__` in a private module hurts, so I kept it. Python leaks enough across module boundaries to warrant some defensive programming.